### ISSUE TITLE
cannot reproduce Again

### ISSUE THREAD

**Issue by betray12138** [Xingyu-Lin/mbpo_pytorch#10]
![image](https://user-images.githubusercontent.com/55013174/227685139-1a94f2bc-e582-447d-b12f-e1b9fa8dfe2c.png)
As the upper figure shows, the walker2D environment's performance can not match the figure you put in the README. What could I do ?

**Comment by Sen1553** [comment]
Hello. Do you tackle it? I met the same  problem. Could you tell me the solution.

**Comment by betray12138** [comment]
> Hello. Do you tackle it? I met the same problem. Could you tell me the solution.

No, I don't. However, I found that the torch version proposed by mbrl repo is more stable, so I choose to use mbrl-lib

**Comment by Sen1553** [comment]
> > Hello. Do you tackle it? I met the same problem. Could you tell me the solution.
> 
> No, I don't. However, I found that the torch version proposed by mbrl repo is more stable, so I choose to use mbrl-lib

Thanks for your suggestion
